src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -284,7 +284,12 @@
 
 @private accessor(accessors)
   @join field : accessors on ""
-    .{@field}()
+    @switch field.kind
+    @case "MEMBER"
+      .{@field.member}()
+    @case "INDEX"
```

Comment:
Maybe add a comment in the template acknowledging that this will throw if `field.index` is out of bounds, but we explicitly don't check.

In the other langs as well.

Reply:
Done